bin/generate-react-native-fixture

```diff
@@ -177,20 +187,20 @@ function configureRN064Fixture(fixtureDir) {
     nodeExecutableAndArgs: ["node", "--openssl-legacy-provider"]
   ]`
 
-  // remove flipper dependencies
+  // remove flipper and associated dependencies
```

Comment:
Why exactly do we do this, again? just to speed up the build process? what do we lose in the process?